### ISSUE TITLE
You can't cast from non-string to string

### ISSUE THREAD

**Issue by cschloer** [BCODMO/frictionless-usecases#14]
[This line](https://github.com/frictionlessdata/tableschema-py/blob/master/tableschema/types/string.py#L20) returns in error in `cast_string` if the value is not an instance of `six.string_types`. `six.string_types` [is just `str`](https://pythonhosted.org/six/#six.string_types), which means that if the value is not already a string, it cannot be cast to a string.

**Comment by roll** [comment]
@cschloer 
It's because it validates data and for logical data representation (in a Python program) only native strings are going to pass validation. E.g. if for a text field your data have some integer values it means that these values are invalid and should be stringified first. 

Any example to take a look at? When it's a problem for a pipeline

**Comment by cschloer** [comment]
@adyork Do you have an example pipeline on hand that displays our need to convert from number to string? Otherwise I will try to send one over when I run into it.

**Comment by adyork** [comment]
OK this is on me, I will find an example

**Comment by adyork** [comment]
So, I took a look at this, and I'm not sure I know of a case where we have needed to turn a non-string into a string.  I have needed to combine two columns (one a string and one a number) to make a cast_id and another time for a unique station_id, but that works currently. 

I was under the impression that it was the other way around that we were having troubles with, taking a string and casting to a number.  @cschloer  wasn't that one of the reasons we were having trouble with our custom lat/lon processor that takes many lat lon formats that are strings and we try to make them into decimal degrees which is a numeric?   Or maybe it was an issue with our round processor?  I forget.

Is this still an issue?  If not we can close this.
